roles/ocm-labels/README.md

```diff
@@ -0,0 +1,105 @@
+ocm-labels
+==========
+
+Manages labels on clusters managed by Red Hat Advanced Cluster Management (RHACM).
+
+
+Requirements
+------------
+
+Clusters defined in the inventory should exist with valid authentication credentials.
+
+
+Role Variables
+--------------
+
+| Variable                | Required           | Default                            | Comments                                 |
+|-------------------------|--------------------|------------------------------------|------------------------------------------|
+| ocm_hub_kubeconfig      | yes                |                                    | Path to kubeconfig for the hub           |
+| ocm_managedcluster_name | yes                |                                    | Cluster for label change                 |
+| ocm_merge_type          | no                 | preserve                           | (More Details Below)                     |
+| ocm_user_labels         | yes                |                                    | Dictonary of desired labels              |
```

Comment:
this make me think that it is a label that indicate the user for the cluster?

ocm_desired_labels might be more descriptive?

Reply:
I agree. ocm_desired_labels is more self-descriptive. I will change it.